Configurable logging docs (#14063)
kyma-project/kyma: docs/01-overview/main-areas/observability/obsv-02-logging-in-kyma.md

title: Logging
---

## Overview

For in-cluster logging, Kyma uses [Loki](https://github.com/grafana/loki), a Prometheus-like log management system.

As a beta feature, Kyma ships the telemetry component, which supports providing your own output configuration for Fluent Bit. With this, you can integrate your own observability systems outside the Kyma cluster with the Kyma backend. If you want to use this feature, [enable the telemetry component](../../../04-operation-guides/operations/obsv-00-enable-configurable-logging.md).

> ## Limitations

Reviewer: Currently we have the following limitations for the configurable logging on the Fluent Bit side:
- Not more than 5 log pipelines when using the production profile or 3 with the evaluation profile.
- Fluent Bit buffers up to 1 GB of logs if a configured output cannot receive logs. The oldest logs are dropped if the limit is reached.
- Each Fluent Bit pod can process up to 10 MB/s of logs for a single log pipeline. With multiple pipelines, the throughput per pipeline is reduced. The used logging backend or performance characteristics of the output plugin might limit the throughput earlier.

> As a beta feature, Kyma ships the [telemetry component](./obsv-04-telemetry-in-kyma.md), which supports providing your own output configuration for Fluent Bit. With this, you can integrate your own observability systems outside the Kyma cluster with the Kyma backend. If you want to use this feature, [enable the telemetry component](../../../04-operation-guides/operations/obsv-00-enable-configurable-logging.md).

Author: ```suggestion
As an alpha feature, Kyma ships the [telemetry component](./obsv-04-telemetry-in-kyma.md), which supports providing your own output configuration for Fluent Bit. With this, you can integrate your own observability systems outside the Kyma cluster with the Kyma backend. If you want to use this feature, [enable the telemetry component](../../../04-operation-guides/operations/obsv-00-enable-configurable-logging.md).
```
